### PR TITLE
🚢 Ship Analyzers

### PR DESCRIPTION
**Pull request by heytherewill**
This:

- Changes the Analyzer project so that the correct .dlls get put in the correct places, thus making the final nupkg behave like an analyzer
- Updates Bang's diagnostic id table and table of shipped analyzers
- Update the relative path in the script that uploads packages to NuGet so that they are all included (given the recent directory changes, not even Bang.dll was being included)

### CODE REVIEW

File: src/Bang.Analyzers/Bang.Analyzers.csproj [hunk 1]
```diff
@@ -1,12 +1,44 @@
...
+    <ItemGroup>
+        <None Include="..\..\LICENSE" Pack="true" PackagePath="" Visible="false"/>
+
+        <!-- Include .dll in the correct folder. -->
+        <None Include="$(OutputPath)/$(AssemblyName).dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+
+        <!-- Files that keep track of what analyzer are/aren't included in releases. -->
+        <None Remove="AnalyzerReleases.Shipped.md" />
+        <None Remove="AnalyzerReleases.Unshipped.md" />
+<!--        <AdditionalFiles Include="AnalyzerReleases.Shipped.md" />-->
```
Comment (isadorasophia) [inline src/Bang.Analyzers/Bang.Analyzers.csproj]: will we need to add `AdditionalFiles` at some point?

File: src/Bang/Bang.csproj [hunk 2]
```diff
@@ -33,8 +33,8 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <None Include="..\LICENSE" Pack="true" PackagePath="" Visible="false"/>
-    <None Include="..\README.md" Pack="true" PackagePath="" Visible="false" />
+    <None Include="..\..\LICENSE" Pack="true" PackagePath="" Visible="false"/>
```
Comment (isadorasophia) [inline src/Bang/Bang.csproj:36]: ops!!! thanks for fixing

**Review by isadorasophia — APPROVED** [review]
Thanks for the fixes too! :shipit:

File: src/Bang.Analyzers/Bang.Analyzers.csproj (hunk 1 above)
Comment (heytherewill) [reply src/Bang.Analyzers/Bang.Analyzers.csproj]: Oh, we do right now 😂

**Pull request merged**